### PR TITLE
[MO] - [tests] -> fix

### PR DESCRIPTION
**Pull request by see-quick**
This PR fixing that tests did not execute on master's branch. 

Signed-off-by: morsak <xorsak02@stud.fit.vutbr.cz>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -202,29 +200,6 @@
 			<type>test-jar</type>
 			<scope>test</scope>
 		</dependency>
-		<dependency>
```
Comment (ppatierno) [inline pom.xml:205]: why all these dependencies are not needed anymore?

File: pom.xml [hunk 2]
```diff
@@ -19,16 +19,14 @@
 		<junit.platform.version>1.5.2</junit.platform.version>
 		<hamcrest.version>2.2</hamcrest.version>
 		<maven-jar-plugin.version>3.1.2</maven-jar-plugin.version>
-		<maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
+		<maven-surefire-plugin.version>2.22.2</maven-surefire-plugin.version>
```
Comment (ppatierno) [inline pom.xml:22]: you mentioned you were using the "milestone" version in the operator project as well, are we sure that using this "old" version doesn't break anything?

File: pom.xml (hunk 1 above)
Comment (see-quick) [reply pom.xml:205]: i have tried execute tests with `3.0.0-M5` but there is some missing dependency which is needed.  In the next PR i will investigate what is the problem with that version but for now we can let this lower version (which is stable)

File: pom.xml (hunk 2 above)
Comment (see-quick) [reply pom.xml:22]: I have tried with `3.0.0-M5` but there is some dependency, which is missing. Anyway, in the next PR I can investigate the problem and change it to this `3.0.0-M5` but for now, we can let it on stable `2.22.2`.

File: pom.xml (hunk 1 above)
Comment (see-quick) [reply pom.xml:205]: These dependencies were needed for that `M5` version

Comment (see-quick) [reply pom.xml:205]: I have tried with 3.0.0-M5 but there is some dependency, which is missing. Anyway, in the next PR I can investigate the problem and change it to this 3.0.0-M5 but for now, we can let it on stable 2.22.2.

File: pom.xml (hunk 2 above)
Comment (see-quick) [reply pom.xml:22]: These dependencies were needed for that M5 version

**Pull request merged**